cnxarchive/sql/schema/triggers.sql

```diff
@@ -1,9 +1,10 @@
 CREATE OR REPLACE FUNCTION update_latest() RETURNS trigger AS '
 BEGIN
-  IF TG_OP = ''INSERT'' AND
+  IF (TG_OP = ''INSERT'' OR TG_OP = ''UPDATE'') AND
           NEW.revised >= ((SELECT revised FROM modules
               WHERE uuid = NEW.uuid ORDER BY revised DESC LIMIT 1)
-              UNION ALL VALUES (NEW.revised) LIMIT 1) THEN
+              UNION ALL VALUES (NEW.revised) LIMIT 1) AND
+          NEW.stateid = 1 THEN
```

Comment:
so, published content will not appear until it has been processed, at all. Hmm, I think that's probably correct.